astropy/table/table.py

```diff
@@ -1903,6 +1903,52 @@ def remove_rows(self, row_specifier):
         if hasattr(self, '_groups'):
             del self._groups
 
+    def remove_na(self):
+        '''
+        Remove rows from a table that has N/A.
+        For masked tables only
+
+        Parameters
+        ----------
+        Masked Table
+
+        Examples
+        --------
+        Create a table with three columns 'a', 'b' and 'c'::
+
+            >>> weather_data = """
+            ...      day,temp,type
```

Comment:
I would prefer to use some astronomy related example, or if must, then at least having the temperature values not in F.

Reply:
I am not familiar with astronomy related examples. If you can suggest some, I can definitely incorporate it in the tests!

Reply:
Oh, using just the pure float number based ones like most of the examples in this file would do. Your example is  fine, just feels weirdly out of place.

Reply:
Should be good now

Reply:
Even though you marked this resolved, my comment hasn't been addressed, the example data still shows a string heavy table rather than what what suggested (my point above was about the content of the table not with the name of the columns).